cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -1,77 +1,72 @@
-    {% load custom_helpers %}
-    {% load humanize %}
+{% load custom_helpers %}
+{% load humanize %}
 
-    {% if features|length > 0 %}
-    <div class="container min-w-full">
-            <div class="table-title table-container-header">
-                <div class="table-container-title">
-                    <i class="bi bi-geo-alt-fill location-tab-icon mr-1"></i>
-                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-icon mr-2">
-                        <!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
-                        <path
-                            d="M416 0c17.7 0 32 14.3 32 32c0 59.8-30.3 107.5-69.4 146.6c-28 28-62.5 53.5-97.3 77.4l-2.5 1.7c-11.9 8.1-23.8 16.1-35.5 23.9l0 0 0 0 0 0-1.6 1c-6 4-11.9 7.9-17.8 11.9c-20.9 14-40.8 27.7-59.3 41.5H283.3c-9.8-7.4-20.1-14.7-30.7-22.1l7-4.7 3-2c15.1-10.1 30.9-20.6 46.7-31.6c25 18.1 48.9 37.3 69.4 57.7C417.7 372.5 448 420.2 448 480c0 17.7-14.3 32-32 32s-32-14.3-32-32H64c0 17.7-14.3 32-32 32s-32-14.3-32-32c0-59.8 30.3-107.5 69.4-146.6c28-28 62.5-53.5 97.3-77.4c-34.8-23.9-69.3-49.3-97.3-77.4C30.3 139.5 0 91.8 0 32C0 14.3 14.3 0 32 0S64 14.3 64 32H384c0-17.7 14.3-32 32-32zM338.6 384H109.4c-10.1 10.6-18.6 21.3-25.5 32H364.1c-6.8-10.7-15.3-21.4-25.5-32zM109.4 128H338.6c10.1-10.7 18.6-21.3 25.5-32H83.9c6.8 10.7 15.3 21.3 25.5 32zm55.4 48c18.4 13.8 38.4 27.5 59.3 41.5c20.9-14 40.8-27.7 59.3-41.5H164.7z"
-                        />
-                    </svg>
-                    DNA Features
-                </div>
-                <div class="absolute right-0">
-                    <span class="search-results-text-small">Download:</span>
-                    <a href="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?accept=text/tab-separated-values{% for feature in feature_types %}&feature_type={{ feature }}{% endfor %}">
-                        <button class="tsv-button search-results-tsv-buttons" id="downloadButton">TSV</button>
-                    </a>
-                    <a href="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?accept=text/tab-separated-values&tsv_format=bed6{% for feature in feature_types %}&feature_type={{ feature }}{% endfor %}">
-                        <button class="tsv-button search-results-tsv-buttons" id="downloadButtonNew">BED6</button>
-                    </a>
-                </div>
+{% if features|length > 0 %}
+    <div class="content-container min-w-full max-w-sm">
+        <div class="table-title flex flex-col sm:flex-row max-w-full min-w-full justify-between items-center">
+            <div class="table-container-title mx-auto">
```

Comment:
Did you mean to remove the icon here entirely, or were you planning on replacing it with a font awesome `i` class?

Also don't forget that the dna_features.html page also uses this partial